app/services/peatio/member_api_v2/client.rb

```diff
@@ -0,0 +1,38 @@
+module Peatio
+  module MemberApiV2
```

Comment:
`MemberAPIv2` and add acronym https://github.com/rubykube/applogic/blob/master/config/initializers/inflections.rb#L19